lib/src/main/java/edu/stanford/futuredata/macrobase/analysis/summary/apriori/APExplanation.java

```diff
@@ -0,0 +1,58 @@
+package edu.stanford.futuredata.macrobase.analysis.summary.apriori;
+
+import edu.stanford.futuredata.macrobase.analysis.summary.Explanation;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * An explanation attempts to summarize the differences between the outliers and inliers.
+ * It is composed of multiple ExplanationResult results, each of which has a matcher which
+ * is an atomic classifier for rows in a dataset.
+ */
+public class APExplanation implements Explanation {
+    private ArrayList<ExplanationResult> results;
+    private double numOutliers;
+    private double numTotal;
+
+    public APExplanation(
+            List<ExplanationResult> results,
+            double numOutliers,
+            double numTotal
+            ) {
+        this.results = new ArrayList<>(results);
+        this.numOutliers = numOutliers;
+        this.numTotal = numTotal;
+    }
+
+    public void sortBySupport() {
```

Comment:
Basically, could do this at construction time, and then have clients resort as needed.